src/main/com/marginallyclever/makelangelo/Translator.java

```diff
@@ -66,7 +66,19 @@ static public void start() {
 	static public boolean isThisTheFirstTimeLoadingLanguageFiles() {
 		// Did the language file disappear?  Offer the language dialog.
 		try {
-			if (doesLanguagePreferenceExist()) {
+			if (doesLanguagePreferenceExist()) {  
+
+                        // Does the language preference have a language name value 
```

Comment:
the formatting does not match the rest of the file: you have added 25 space char instead of 4 tabs.

Reply:
> the formatting does not match the rest of the file: you have added 25 space char instead of 4 tabs.

Sorry for the spaces instead of the tabs (bad configuration of my editor ...) ... by the time I change, the request has been merged.
Should I redo a PR?